GuysNight.LethalCompanyMod.BalancedItems/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Version 1.6.0
+
+1. You can now modify the purchase price of any item bought from the terminal. Custom items from other mods are
```

Comment:
This was reverted from this PR. Will be in the next PR. But didn't want to deal with splicing it out for now.